### PR TITLE
makeybutton: better algorithm implementation

### PR DESCRIPTION
**Pull request by deadprogram**
This PR modifies the algorithm to better match the original.

### CODE REVIEW

**Comment by ysoldak** [comment]
How do you know it matches original better, trial and error or there is code available to compare with?

File: makeybutton/button.go [hunk 1]
```diff
@@ -49,7 +54,7 @@ func NewButton(pin machine.Pin) *Button {
 
 // Configure configures the Makey Button pin to have the correct settings to detect touches.
 func (b *Button) Configure() error {
-	// Note that we have to first turn on the pullup, and then turn off the pullup,
+	// Note that on AVR we have to first turn on the pullup, and then turn off the pullup,
```
Comment (ysoldak) [inline makeybutton/button.go:60]: Same holds for non-AVR? Or this driver for AVR only?

Comment (deadprogram) [reply makeybutton/button.go:60]: It is only "required" for AVR, but does not do anything bad on the other platforms I tested (mostly rp2040).

**Comment by deadprogram** [comment]
See https://github.com/sparkfun/MaKeyMaKey/blob/master/firmware/Arduino/makey_makey/makey_makey.ino for the "original".

I will add that link to the source for clarity.

**Comment by ysoldak** [comment]
And not approve, but LGTM :D

**Pull request merged**